### ISSUE TITLE
[README.md] instructions to run the Docker image give error

### ISSUE THREAD

**Issue by bhavin192** [transtats/transtats#70]
##### ISSUE TYPE
 - Documentation Report

##### SUMMARY
If I follow the instructions given for running transtats using Docker image, 
and pull the image `transtats/transtats`
The given command to run the image gives error as there is no image called `transtats`.
Given instructions will work if someone builds that image as `$ sudo docker build -t transtats .`

##### STEPS TO REPRODUCE
```
$ sudo docker pull transtats/transtats
$ sudo docker run -d --name container_name -p 8080:8015 transtats
```
##### ACTUAL RESULTS

```shell
[bhavin192@test-vm ~]$ sudo docker run -d --name container_name -p 8080:8015 transtats                                                              
Unable to find image 'transtats:latest' locally
docker: Error response from daemon: repository transtats not found: does not exist or no pull access.
See 'docker run --help'.
```

**Comment by bhavin192** [comment]
I'm working on a patch to fix this.

**Comment by sundeep-co-in** [comment]
thanks @bhavin192

**Comment by bhavin192** [comment]
Closing as the PR #71 is merged by b6b06fae25f11834ae6a0945b167298d319a377b
